server/main.lua

```diff
@@ -150,6 +150,15 @@ RegisterNetEvent('hospital:server:RevivePlayer', function(playerId)
 	TriggerClientEvent('hospital:client:Revive', patient.PlayerData.source)
 end)
 
+---@param playerId number
+RegisterNetEvent('hospital:server:putPlayerInBed', function(playerId, hospitalName, bedIndex)
+	if GetInvokingResource() then return end
+	local patient = exports.qbx_core:GetPlayer(playerId)
+
+	if not patient then return end
+	TriggerClientEvent('qbx_ambulancejob:client:putPlayerInBed', patient.PlayerData.source, hospitalName, bedIndex)
```

Comment:
No need to get the player object just for the source
```suggestion
	TriggerClientEvent('qbx_ambulancejob:client:putPlayerInBed', playerId, hospitalName, bedIndex)
```